docs/toc.md

```diff
@@ -161,23 +164,34 @@
 ## [WeakEventListener](helpers/WeakEventListener.md)
 ## [ViewportBehavior](helpers/ViewportBehavior.md)
 
+# Services
+## [Facebook Service](services/Facebook.md)
+## [LinkedIn Service](services/Linkedin.md)
+## [Microsoft Graph Service](services/MicrosoftGraph.md)
+### [GraphLogin (WinForms)](services/GraphLogin.md)
+## [Microsoft Translator Service](services/MicrosoftTranslator.md)
+## [OneDrive Service](services/OneDrive.md)
+## [Twitter Service](services/Twitter.md)
+## [Weibo Service](services/Weibo.md)
+
+# Parsers
+## [MarkdownParser](parsers/MarkdownParser.md)
+## [RSSParser](parsers/RSSParser.md)
+## [Win2d Path Mini Language Parser](parsers/CanvasPathGeometry.md)
+## [Win2d Path Mini Language](parsers/Win2d_Path_Mini_Language.md)
```

Comment:
We may just want to put these under Helpers for now or rename Brushes to Media. @Sergio0694 did we need to docs for new VisualEffects Factory stuff somewhere?

Reply:
We have that in a sample page but don't think we have that in the actual docs yet. I can make a PR to add a page with a small example to showcase how to use it though 🙂